File: Unidad 5 C#/Programa8U5.cs
using System;

namespace Programa8u5
{
    internal class Programa8U5
    {
        static void Main(string[] args)
        {
            Console.Write("Ingrese la cantidad de veces que desea lanzar el dado: ");
            int vecesLanzamiento = Convert.ToInt32(Console.ReadLine());

            Console.WriteLine("Resultados de los lanzamientos:");

            for (int i = 0; i < vecesLanzamiento; i++)
            {
                int resultado = LanzarDado();
                Console.WriteLine("Lanzamiento " + (i + 1) + ": " + resultado);
            }
        }

        static int LanzarDado()
        {
            Random random = new Random();
            return random.Next(1, 7);
        }
    }
}

File: Unidad 5 Java/Programa8U5.java
package com.mycompany.programa8u5;
import java.util.Random;
import java.util.Scanner;

public class Programa8U5 {

    public static void main(String[] args) {
          Scanner sc = new Scanner(System.in);

        System.out.print("Ingrese la cantidad de veces que desea lanzar el dado: ");
        int vecesLanzamiento = sc.nextInt();

        System.out.println("Resultados de los lanzamientos:");

        for (int i = 0; i < vecesLanzamiento; i++) {
            int resultado = lanzarDado();
            System.out.println("Lanzamiento " + (i + 1) + ": " + resultado);
        }
    }

    static int lanzarDado() {
        Random random = new Random();
        return random.nextInt(6) + 1;
    }
}
